tests/testthat/test-mock.r

```diff
@@ -116,9 +116,11 @@ test_that("can mock hidden functions", {
 })
 
 test_that("can mock if package is not loaded", {
-  if ("package:devtools" %in% search())
-    skip('devtools is loaded')
-  with_mock(`devtools::add_path` = identity, expect_identical(devtools::add_path, identity))
+  if ("package:boot" %in% search())
+    skip('boot is loaded')
+  if (!length(find.package("boot", quiet=TRUE)))
```

Comment:
But then you'll have to get that on CRAN?  Why can't we go back to using devtools which is already in suggests?